app/presenters/default_deadline_date_time.rb

```diff
@@ -0,0 +1,13 @@
+# used for computing the default deadline with an offset
+class DefaultDeadlineDateTime
+  attr_reader :dc_time
+
+  def self.parse(start_time_str, day_offset)
+    time = DcTimePresenter.parse(start_time_str)
+    new(time, day_offset)
+  end
+
+  def initialize(start_time, day_offset)
+    @dc_time = day_offset.business_days.after(DcTimePresenter.new(start_time).convert)
```

Comment:
the model you have here is a pattern I've  seen done in the past (have a class method instantiate) but in this codebase we aren't doing much of it so I could go either way...